Automattic/vip-cli
# WP CLI: Fix single quotes bug

## Description

Fixing a bug where single quote arguments are not interpreted properly in VIP CLI.
Verifying that the requoted command gets passed to the API.
WIP

Fixes #365 

## Steps to Test

WIP

1. Check out PR.
2. Run `npm run build`
3. Run `./dist/bin/vip-wp.js`
4. Run a command that involves single quotes (e.g.- `wp option update blogdescription 'Single quotes'`
5. Verify there's no error
